dev-docs/bidders/ozone.md

```diff
@@ -11,6 +11,7 @@ gvl_id: 524
 deals_supported: true
 schain_supported: true
 coppa_supported: true
+gpp_supported: true
```

Comment:
Please replace this with `gpp_sids` meta data. See https://docs.prebid.org/dev-docs/modules/consentManagementGpp.html#bidder-adapter-gpp-integration

> If you are submitting changes to an adapter to support GPP, please also submit a PR to the [docs repo](https://github.com/prebid/prebid.github.io) to add the `gpp_sids` meta data with a comma separated list of section names (tcfeu, tcfca, usnat, usstate_all, usp) to your respective page in the [bidders directory](https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders). This will ensure that your adapter’s name will automatically appear on the list of adapters supporting GPP.